graphics/apitrace/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem             1.0
+
```

Comment:
no newline please

Reply:
The newline at line 2 wasn't intentional, didn't notice. The one BELOW `PortSystem` is correct though, yes?